motr/idx_dix.c

```diff
@@ -1002,11 +1016,18 @@ static void dix_index_create_ast(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 	struct m0_dix_req       *dreq = &dix_req->idr_dreq;
 	struct m0_dix            dix;
 	int                      rc;
+	uint32_t		 flags = 0;
 
 	M0_ENTRY();
 	dix_build(oi, &dix);
+
+	if (oi->oi_flags & M0_OIF_CROW)
```

Comment:
Can you add a helper function to set the flags, as it used at many places ?

Reply:
Added new helper function dix_idx_set_cas_flags().